server/src/main/java/org/opensearch/index/search/QueryStringQueryParser.java

```diff
@@ -472,6 +493,7 @@ private Query getRangeQuerySingle(
 
     @Override
     protected Query handleBareFuzzy(String field, Token fuzzySlop, String termImage) throws ParseException {
+        onDiscoveredField(field);
```

Comment:
Not needed, `getFuzzyQuery` discovers fields

Reply:
done